### ISSUE TITLE
"ad" is not defined

### ISSUE THREAD

**Issue by ncfavier** [UniMath/SymmetryBook#194]
$\mathrm{ad}$ is used here:

https://github.com/UniMath/SymmetryBook/blob/33011ebf1fc139cc6f17ab45b5f8db7ae4b4a00a/group.tex#L2356

But is not defined. I guess it means conjugation?

**Comment by pierrecagne** [comment]
Indeed, this 'ad' has been deprecated. It should be $\Omega \Bf$ now according to Def 4.4.5.
